WordPress/WordPressTest/SearchMenuItemCreatorTests.swift

```diff
@@ -23,7 +23,7 @@ final class SearchMenuItemCreatorTests: XCTestCase {
     func testItemCreatorReturnsItemWithExpectedTitle() {
         let item = creator!.menuItem()
 
-        XCTAssertEqual(item.title, TestConstants.title)
+        XCTAssertEqual(item.title, String(format: TestConstants.title))
```

Comment:
I'm curious. Why did you use `String(format:)` here?

Reply:
when using a variable, you have to use format in order for the localization to work

Reply:
That's interesting. I tried removing this earlier and the test still passed. 🤷‍♂️